data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
 		"drag" 16.1
 		"heat dissipation" .4
 		"fuel capacity" 700
-		"cargo space" 530
-		"outfit space" 740
-		"weapon capacity" 300
-		"engine capacity" 180
+		"cargo space" 240
+		"outfit space" 720
+		"weapon capacity" 280
+		"engine capacity" 160
```

Comment:
Good change. Now the 82-outfit human atomic thruster and the Bufaer Hai thruster will go snugly (158 space) into the engine capacity.